packages/edit-post/src/hooks/use-load-entity-record.js

```diff
@@ -16,9 +15,9 @@ import { addQueryArgs, getQueryArgs, removeQueryArgs } from '@wordpress/url';
  * @param {string} initialPostType The post type of the post when the editor loaded.
  *
  * @return {Object} An object containing the `currentPost` variable and
- *                 `getPostLinkProps` and `goBack` functions.
+ *                 `onSelectEntityRecord` and `goBack` functions.
  */
-export default function usePostHistory( initialPostId, initialPostType ) {
+export default function useLoadEntityRecord( initialPostId, initialPostType ) {
```

Comment:
I did initially have this called `useSelectEntityRecord` but is caused a linting error about needed to pass in a callback - the linter was confusing it with `useSelect` I think, and likely that naming would be confusing to others as well.